toolkit/themes/shared/design-system/build/css/tokens-shared.css

```diff
@@ -0,0 +1,175 @@
+  --color-cyan-50: #00ddff;
+  --color-gray-05: #fbfbfe;
+  --color-gray-50: #bfbfc9;
+  --color-gray-60: #8f8f9d;
+  --color-gray-70: #5b5b66;
+  --color-gray-80: #23222b;
+  --color-gray-90: #1c1b22;
+  --color-gray-100: #15141a;
+  --color-green-05: #d8eedc;
+  --color-green-30: #4dbc87;
+  --color-green-50: #017a40;
+  --color-green-80: #004725;
+  --color-red-05: #ffe8e8;
+  --color-red-20: #ff9aa2;
+  --color-red-30: #f37f98;
+  --color-red-50: #d7264c;
+  --color-red-80: #690f22;
+  --color-white: #ffffff;
+  --color-yellow-05: #ffebcd;
+  --color-yellow-30: #e49c49;
+  --color-yellow-50: #cd411e;
+  --color-yellow-80: #5a3100;
+
+  /* Application tokens */
+  /** Border **/
+  --border-radius-circle: 9999px;
+  --border-radius-medium: 8px;
+  --border-radius-small: 4px;
+  --border-width: 1px;
+
+  /** Box **/
+  --box-background-color: light-dark(var(--color-white), var(--color-gray-80));
+  --box-shadow-10: 0 1px 4px var(--color-black-a10);
+
+  /** Color **/
+  --color-background-critical: light-dark(var(--color-red-05), var(--color-red-80));
+  --color-background-information: light-dark(var(--color-blue-05), var(--color-blue-80));
+  --color-background-success: light-dark(var(--color-green-05), var(--color-yellow-80));
+  --color-background-warning: light-dark(var(--color-yellow-05), var(--color-blue-80));
+  --color-error-outline: light-dark(var(--color-red-50), var(--color-red-20));
+
+  /** Font weight **/
+  --font-weight: normal;
+  --font-weight-bold: 600;
+
+  /** Focus outline **/
+  --focus-outline: var(--focus-outline-width) solid var(--focus-outline-color);
+  --focus-outline-color: var(--color-accent-primary);
+  --focus-outline-inset: calc(-1 * var(--focus-outline-width));
+  --focus-outline-offset: 2px;
+  --focus-outline-width: 2px;
+
+  /** Icon **/
+  --icon-color: light-dark(var(--color-gray-70), var(--color-gray-05));
+  --icon-color-critical: light-dark(var(--color-red-50), var(--color-red-30));
+  --icon-color-information: light-dark(var(--color-blue-50), var(--color-blue-30));
+  --icon-color-success: light-dark(var(--color-green-50), var(--color-green-30));
+  --icon-color-warning: light-dark(var(--color-yellow-50), var(--color-yellow-30));
+
+  /** Input **/
+  /*** Button ***/
+  --button-border-radius: var(--border-radius-small);
+  --button-font-weight: var(--font-weight-bold);
+  --button-line-height: var(--line-height-small);
+  --button-min-height: var(--size-item-large);
+
+  /*** Text ***/
+  --input-text-line-height: var(--button-line-height);
+  --input-text-min-height: var(--button-min-height);
+
+  /** Link **/
+  --link-focus-outline-offset: 1px;
+
+  /** Text **/
+  --text-color-deemphasized: color-mix(in srgb, currentColor 60%, transparent);
+  --text-color-error: light-dark(var(--color-red-50), var(--color-red-20));
+
+  /** Size **/
+  --size-item-large: 32px;
+  --size-item-medium: 28px;
+  --size-item-small: 16px;
+
+  /* Unspecified */
+  --spacing-small: 0.5rem;
```

Comment:
I don't think we have this token in central anymore (https://searchfox.org/mozilla-central/search?q=--spacing-&path=&case=false&regexp=false)